src/odemis/gui/cont/tabs.py

```diff
@@ -2762,6 +2783,18 @@ def remove_overview_stream(self, overview_stream):
         except AttributeError:  # No overview view
             pass
 
+    def remove_overview_streams(self, streams):
```

Comment:
This is very similar to `remove_overview_stream()`. That function is called just once. I'd suggest to remove that function, and update the caller to do `remove_overview_streams([s])`. This way you'll avoid having twice almost the same code.